### ISSUE TITLE
Estimation of non-sampled hauls in onboard sampling

### ISSUE THREAD

**Issue by luciazarauz** [ices-tools-dev/RDBES#208]
Dear core group,

When we sample trawlers on board, each trip lasts an average of one wek, and the observer cannot sample all the hauls in a trip. In our data base, we make imputations to estimate the catch of those non-sampled hauls, using the most similar hauls in the same trip (expert knowledge). 

We are registering which are the hauls that we use for the imputations, and we plan to use this information to replicate the estimates using RDBES data by importing an auxiliary file in the estimation process. 

However,  to estimate the catch of the non-sampled hauls, we also need the total weight of the haul (all species together), which is an information collected by the observer, also for the non-sampled hauls. 

Shall we include the total weight of the haul also as auxiliary informtion? or is it a way to report it using the RDBES format? 
We are using hierarchy 3.

Thank you very much

Best regards

**Comment by HenrikK-N** [comment]
It is suggested to use the auxiliary field FOauxiliaryVariableTotal in the FO table for the total weight of a haul, then FOauxiliaryVariableName could be 'TotalWeightOfHaul'. The field  FOauxiliaryVariableUnit could contain 'kg'. Could that be a solution?

**Comment by luciazarauz** [comment]
yes, that could work. 

At the moment we have an auxiliary file with the information about the hauls that we use for imputations to estimate non sampled hauls (for example, FOnumber = 3 was not sampled, and we use FOnumber 1 in a 50%, FOnumber = 2 in a 25% and FOnumber = 4 in a 25% to estimate it)
I find it difficult to include also this nformation in the format. It is very specific... (and we may change the method in the future)
Is it OK if we take this information from an auxiliary file?

Thanks

**Comment by HenrikK-N** [comment]
Yes, it is okay to upload this information in an auxiliary file in TAF and use that.
